api/Node.json

```diff
@@ -489,10 +489,11 @@
               "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": "≤12.1",
+              "version_removed": "15"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": false
```

Comment:
If `node.getFeature()` was in Opera for desktop then it was almost certainly in Opera for Android too.